src/model/CoffeeShop.kt

```diff
@@ -0,0 +1,19 @@
+/**
+ * Defines a CoffeeShop object, containing a CoffeeShop name, its coordinates and a given user's coordinates
+ *
+ * created by AlexandraV on 12/26/2018
+ */
+data class CoffeeShop(val shopName: String, val shopCoordinate: Coordinate, val userCoordinate: Coordinate) : Comparable<CoffeeShop> {
```

Comment:
What if you want to compare the same coffee shop to more than one user? How would the CoffeeShop class look like then?